gapis/service/path/path.proto

```diff
@@ -330,7 +330,13 @@ message Mesh {
 
 // MeshOptions provides parameters for the mesh returned by a Mesh path resolve.
 message MeshOptions {
-    bool faceted = 1; // If true then normals are calculated from each face.
+    // If true then normals are calculated from each face.
+    bool faceted = 1;
+    // If true, the actual data will be missing from the vertex.Buffers.
+    bool exclude_data = 2;
+    // Hints that override the semantic guessing. Maps vertex attribute names to
+    // their intentended semantic type.
+    map<string, vertex.Semantic.Type> vertex_semantics = 3;
```

Comment:
We've intentionally avoided proto's map to date. While I am happy to revisit that decision, I'd prefer consistency for now.

Reply:
Do you think that name to type is the right association here?
I was thinking that stream index to type might be more explicit and less error prone (like when there is no name).